### PR TITLE
🌐 [localization]: Add snarky translations

### PR DESCRIPTION
**Pull request by vegerot**
Co-Authored-By: "Matthew \"strager\" Glazar" <strager.nds@gmail.com>

### CODE REVIEW

**Comment by vegerot** [comment]
we should credit @EmptyGroceryBag too prob

**Comment by vegerot** [comment]
and chat

**Comment by vegerot** [comment]
Co-Authored-By: "Twitch \"Kappa\" Chat" <support@twitch.com>

File: po/en_US@snarky.po [hunk 1]
```diff
@@ -1,41 +1,26 @@
...
-# quick-lint-js is distributed in the hope that it will be useful,
-# but WITHOUT ANY WARRANTY; without even the implied warranty of
-# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-# GNU General Public License for more details.
-#
-# You should have received a copy of the GNU General Public License
-# along with quick-lint-js.  If not, see <https://www.gnu.org/licenses/>.
-#
+## Copyright (C) 2020  Matthew "strager" Glazar
+## See end of file for extended copyright information.
```
Comment (vegerot) [inline po/en_US@snarky.po:2]:
```suggestion
# Copyright (C) 2020  Matthew "strager" Glazar
# See end of file for extended copyright information.
```

File: po/en_US@snarky.po [hunk 2]
```diff
@@ -1127,4 +1119,7 @@ msgstr "expected {1:singular}"
 
 #: test/test-vim-qflist-json-diag-reporter.cpp
 msgid "here"
-msgstr "here"
+msgstr "there 👈"
+
+## Copyright (C) 2020  Matthew "strager" Glazar
+## See top of file for extended copyright information.
```
Comment (vegerot) [inline po/en_US@snarky.po:1125]:
```suggestion
# Copyright (C) 2020  Matthew "strager" Glazar
# See top of file for extended copyright information.
```

**Comment by strager** [comment]
I fixed the emoji issue in commit 1a435c6071f4a4697f58fb9970dc4ab0d33f70d9.

I cleaned up and merged this PR as commit 7c3da5acd9f908796e09f9ba5460daab01eb95f1.

Thanks for helping with this patch!

**Pull request closed without merging**